docker/from-packages/scripts/start-collabora-online.pl

```diff
@@ -34,9 +35,42 @@ ()
 
                 $output .= "                </group>\n";
             }
+        } elsif (/^domain/) {
+            my $value = $ENV{$_};
+            $message .= "WARNING: The 'domain=$value' is deprecated.\n For your convenience, we interpret it as the following:\n";
+            my @hosts = split('\|', $value);
+            if (@hosts) {
+                my $i = 0;
+                foreach (@hosts) {
+                    $i++;
+                    $message .= "   aliasgroup".$i."=https://$_:443\n";
+                    $output .= "                <group>\n";
+                    $output .= "                    <host desc=\"hostname to allow or deny.\" allow=\"true\">https://$_:443</host>\n";
+                    $output .= "                </group>\n";
+                }
+                if (@hosts >= 2) {
+                    $message .= "This means that people from $hosts[0] will not be able to access documents from ";
+                    for ($b = 1; $b < @hosts; $b = $b + 1)
+                    {
+                        $message .= "$hosts[$b], ";
+                    }
+                    $message .= " and vice versa";
+                }
+                $message .= ". If you want to allow the access instead, use this configuration instead:\n     aliasgroup1=";
+                $i = 0;
+                foreach(@hosts) {
```

Comment:
You can use join() instead of this foreach()

Reply:
In this case join() does not work